tests/FeeSharingProxyTest.js

```diff
@@ -469,7 +447,8 @@ contract("FeeSharingProxy:", (accounts) => {
             let feeSharingProxyBalance = await loanTokenWrbtc.balanceOf.call(
                 feeSharingProxy.address
             );
-            expect(feeSharingProxyBalance.toString()).to.be.equal(feeAmount.toString());
+            // feeSharingProxy no longer provide the liquidity to lending pool.
```

Comment:
```suggestion  
            // feeSharingProxy no longer provides the liquidity to   
```

Reply:
Updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/458/commits/409546f296d6768b74e97f31bdaec2a50b341684#diff-ded3c1c096857530aa79f9578613abea720bb45f154b870c764c559e976104ed)